shell/platform/darwin/ios/framework/Source/FlutterTextureRegistryRelay.mm

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterTextureRegistryRelay.h"
+
+#include "flutter/fml/logging.h"
+
+@implementation FlutterTextureRegistryRelay : NSObject
+
+#pragma mark - FlutterTextureRegistry
+
+- (instancetype)initWithParent:(NSObject<FlutterTextureRegistry>*)parent {
+  self = [super init];
+  if (self != nil) {
```

Comment:
optional nits:
```suggestion
if (self = [super init]) {
```

this is following the pattern in the objc document: https://developer.apple.com/documentation/objectivec/nsobject/1418641-init?language=objc

Reply:
done